### PR TITLE
chore: bump Angular 19

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## Description

Migrate LF to Angular 19 and related packages (storybook, jest, eslint)

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3262/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3262/storybook

File: .eslintrc
Comment (shajz) [inline .eslintrc:1]: Reminder for later: we could run the eslint migration on LF :D

File: packages/ng/dialog/dialog-routing/dialog-routing.component.ts [hunk 1]
```diff
@@ -87,15 +87,23 @@ export class DialogRoutingComponent<C> implements OnInit {
 
 	#ref?: LuDialogRef<C>;
 
-	async ngOnInit(): Promise<void> {
+	ngOnInit(): void {
```
Comment (shajz) [inline packages/ng/dialog/dialog-routing/dialog-routing.component.ts:90]: isn't this breaking?

File: packages/ng/icon/icon.component.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@use '@lucca-front/icons/src/main';
+// @use '@lucca-front/icons/src/main';
```
Comment (shajz) [inline packages/ng/icon/icon.component.scss:1]: delete?

File: packages/ng/styles/components/cdk/_overlay.scss [hunk 3]
```diff
@@ -30,6 +30,10 @@
 	position: absolute;
 	pointer-events: auto;
 	z-index: 1000;
+
+	> * {
+		flex-grow: 1;
+	}
```
Comment (shajz) [inline packages/ng/styles/components/cdk/_overlay.scss:36]: Not sure about this but shouldn't this be in a dedicated PR?

Comment (Supamiu) [reply packages/ng/styles/components/cdk/_overlay.scss:36]: This one is due to a change in cdk that adds flex to the default overlay container, this SCSS is here as a workaround for this.

File: packages/ng/dialog/dialog-routing/dialog-routing.component.ts (hunk 1 above)
Comment (GuillaumeNury) [reply packages/ng/dialog/dialog-routing/dialog-routing.component.ts:90]: It could break some manual `ngOnInit().then` calls but it would be catched by TS compilation

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3262/storybook

File: packages/ng/icon/icon.component.scss (hunk 2 above)
Comment (GuillaumeNury) [reply packages/ng/icon/icon.component.scss:1]: Damn, I forget about this one:

```bash
✘ [ERROR] Could not resolve "../../icons/src/commons||file://cdn.lucca.fr/lucca-front/icons/font/lucca-icons.woff" [plugin angular-css-resource]

    icon/icon.component.scss:14:91:
      14 │ ...f2"), url("../../icons/src/commons||file://cdn.lucca.fr/lucca-f...
         ╵              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  Preprocessor stylesheets may not show the exact file location of the error.
```

It should be fixed before merging this PR

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3262/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3262/storybook

**Review by vvalentin-lucca — APPROVED** [review]

**Review by jeremie-lucca — APPROVED** [review]
